humans/forms.py

```diff
@@ -21,16 +22,61 @@ class Meta:
             "fingerprint",
             "server_signed",
             "timezone",
-            "language",
+            "language"
         ]
+
         widgets = {
             'keyserver_url': forms.TextInput(attrs={'placeholder': _("https://example.com/key.asc")}),
             'public_key': forms.Textarea(attrs={'placeholder': _("-----BEGIN PGP PUBLIC KEY BLOCK-----\nVersion: SKS 1.1.1\n<PGP KEY>\n-----END PGP PUBLIC KEY BLOCK-----")})
         }
 
+    current_password = forms.CharField(label=_('Current password'),
+                                   required=False,
+                                   widget=forms.PasswordInput)
+    new_password1 = forms.CharField(label=_('New password'),
+                                    required=False,
+                                    widget=forms.PasswordInput)
+    new_password2 = forms.CharField(label=_('New password confirmation'),
+                                    required=False,
+                                    widget=forms.PasswordInput)
+
     def __init__(self, *args, **kwargs):
+        # Flag to let the save method know when to call set_password
+        self.change_password = False
         self.pub_key = None
-        return super().__init__(*args, **kwargs)
+        return super(UpdateUserInfoForm, self).__init__(*args, **kwargs)
```

Comment:
Why this change? This is a Python 3 project, there is no need to call super using the old "style".